### PR TITLE
[INT-185] Replace Unlicense with CC0-1.0

### PR DESCRIPTION
**Pull request by kirelagin**
CC0-1.0 is considered to be a better Public Domain dedication than
Unlicense, so it is preferable.

https://softwareengineering.stackexchange.com/q/147111/218480

Use CC0-1.0 for files that are likely not copyrightable:

* The ones that we previously used Unlicense for.
* Use it in package.yaml too.

### CODE REVIEW

**Comment by kirelagin** [comment]
I have also updated the [FAQ in Notion](https://www.notion.so/serokell/FAQ-8da3477b8b7c4faaab6460ff38fc8929) to explicitly give `package.yaml` as an example of a not-copywritable file.

File: .reuse/dep5 [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 Files: templates/.github/*
-Copyright: 2019 Serokell <https://serokell.io>
-License: Unlicense
+Copyright: 2020 Serokell <https://serokell.io>
+License: CC0-1.0
 
 Files: templates/reuse/dep5
-Copyright: 2019 Serokell <https://serokell.io>
-License: Unlicense
+Copyright: 2020 Serokell <https://serokell.io>
```
Comment (gromakovsky) [inline .reuse/dep5:11]: Isn't it more correct to change 2019 to 2019-2020 rather than just 2020?

**Review by gromakovsky — APPROVED** [review]
I have one question/suggestion, but I suppose nobody really cares about these years anyway and for non-copyrightable files people care about them even less by another order of magnitude, so I approve.

File: .reuse/dep5 (hunk 1 above)
Comment (kirelagin) [reply .reuse/dep5:11]: Yeah, you are right.

**Comment by gromakovsky** [comment]
I am not sure if all these files with 2018 or 2019 year weren't changed in 2020, but I don't want to check it and don't really care, and I've approved already.

**Comment by gromakovsky** [comment]
Gonna squash and sign commits here and merge this one as well.

**Pull request merged**